app/presenters/organisations/featured_news_presenter.rb

```diff
@@ -0,0 +1,48 @@
+module Organisations
+  class FeaturedNewsPresenter
+    attr_reader :org
+
+    def initialize(organisation)
+      @org = organisation
+    end
+
+    def first_featured_news
+      main_story = featured_news([org.ordered_featured_documents.first])
+      main_story = main_story[0]
+      main_story[:large] = true
+      main_story
+    end
+
+    def remaining_featured_news
+      return featured_news(org.ordered_featured_documents[1, 100]) if org.is_news_organisation?
```

Comment:
Is `[1, 100]` being used to remove the first item? If so, you could use `.drop(1)` instead.